src/main/java/com/powsybl/openloadflow/network/LfNetwork.java

```diff
@@ -447,10 +465,10 @@ public void reportBalance(Reporter reporter) {
             this, activeGeneration, activeLoad, reactiveGeneration, reactiveLoad);
     }
 
-    public void fix(boolean minImpedance, boolean dc, double lowImpedanceThreshold) {
+    public void fix(boolean minImpedance, double lowImpedanceThreshold) {
         if (minImpedance) {
             for (LfBranch branch : branches) {
-                branch.setMinZ(dc, lowImpedanceThreshold);
```

Comment:
I think we should not do this here at a post loading fix. z cut could be done in a cleaner way at branch creation.

Reply:
It has been designed like that because min z what dependent of the AC or DC context. Now, as we store both, we can do that at branch creation. Do you want me to do this?